db/ddl.sql

```diff
@@ -100,13 +100,14 @@ GRANT ALL ON TABLE award TO "tttAdmin";
 /********************************************
   log
  *******************************************/
-DROP TABLE IF EXISTS log CASCADE;
+DROP TABLE IF EXISTS log;
 CREATE TABLE log (
   "id" BIGINT NOT NULL PRIMARY KEY,
   "user_account_id" INT NOT NULL REFERENCES user_account(id),
   "controller_class" VARCHAR(255) NULL,
-  "operation" CRUD_OPERATION NOT NULL,
-  "property" VARCHAR(255) NOT NULL,
+  "subject_id" BIGINT NOT NULL,
+  "operation" varchar(50) NULL,
```

Comment:
I like this change. I was wondering how we would handle it with the ENUM. Just using a string is a nice simplification.